build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.1.2'
+	id 'io.spring.dependency-management' version '1.1.2'
+}
+
+group = 'com.valmiki'
+version = '0.0.1-SNAPSHOT'
+
+java {
+	sourceCompatibility = '17'
+}
+
+configurations {
```

Comment:
Refer to the vidyarthi service, and add missing dependencies and tasks
Also make sure to run clean build in local before pushing

Reply:
fixed